hm_pyhelper/hardware_definitions.py

```diff
@@ -34,6 +34,7 @@ def is_raspberry_pi():
         'FRIENDLY': 'Nebra Indoor Hotspot Gen 1',
         'APPNAME': 'Indoor',
         'SPIBUS': 'spidev1.2',
+        'ECCBUS': 'i2c-1',
```

Comment:
I think `ECC_I2C_BUS` would be better. Our current keys like `SPIBUS` are not as clear as they could be.

Reply:
Yup, just went along with the `SPIBUS` naming convention here.  Also used only a device name for coherence.